src/electron/device-connect-view/Styles/deviceconnectview.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/common.scss';
+
+body {
+    -webkit-app-region: drag;
```

Comment:
This lets you click and drag the window around since the system title bar is disabled.